dist/routable-component.js

```diff
@@ -209,8 +209,8 @@ var routableComponent =
 	     */
 	
 	  }, {
-	    key: 'unregister_component',
-	    value: function unregister_component(name) {
+	    key: 'unregisterComponent',
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - Expected indentation of 2 space characters but found 0. (indent)

<!-- Comment by [SideCI](https://sideci.com) -->